playbooks/deploy.yml

```diff
@@ -60,41 +60,58 @@
+      pulp_python:
+        ssh_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDOOwu9Z/TldohLPMmdnLXEc9k4hTTLJdVzvtYi+udphv86MESpWtF87RXJ8suHVJd6EDi3cLtd0yy5y1DRCCCF4JnY93WDmWQVA7VEvwcryOa5qRtg2l/4VmFEWyA+B+QpYBrKgNlEh+NoyOjrJusI/pc3GPAAsx12hZVTu+kgLWer2QLVGg0b8EPxyUNVddUyishJlidzW79xGSoPL1+LhAJgAfAQejluZbzN7LiK5+GiK2vLRdddMpRVw04/JCqPIbmpzPSnzT6iIvw9UUQWS/pdXexp9IlZjCbECojwOCaN8wcJZ8tHIaLOwDKoMfxm5W0O5czMWcG/03gOi/u9Ny0pEtqI4K2MnoyIY+uBszurhBn248TrPoAeLy8Ate7rZ/BFdD3MJXD0oTV+mY/CTM33HgksaBkyzgyYsdJdysrw94eljweGZE+z6euZyEJsv+goGD65hgNBpxWXq/3HY3YUGazj4LHkbFPVw/pCrkffEekWYPchCPPAxtwEvoc= dkliban@localhost.localdomain"
+      pulp_rpm:
+        ssh_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDFErIHOwLyM/uLv8qsB3jeBwOQF43gtTTKypIAVJZKaSsHAPYvepsqP04GC4L+Q0TNy5oIMoIy/IxJsHZhRQWVEyPC5AOFBlSxf9zuLRop/OkMaP8OsZAtBvvArqQa7n+FGKaLonUZrHpQ+fH8xsmiK9MoNHyKhz58Wb6yyKL0SqTlQz1tulVaD57aiqSvSAqCvL6cyF9LSCL5QZPsJJL8rY2jqAYnYpWkhznesWqafE1WWTBSptdOq0ojSf+WwBknyTHp4/as8ESHJHyDXSjesMdItYnNLO9khSh24g/mCLE5XTpIIbda7BEDYtCH5ssYO4tBIjmNoCamLMBmCseGyVPzhoiHbiRGaM0oB76cm72EOdlahmDP77/SSSg4yI2kMieFXZ2lRqWF6qaJC1WoDY0zFZRNMoAZ+ooARvmFf1qsWRGtfaZ3awJpGgZWBb3totXXOJpvUQRRd2TxJypXxDYfxWOxcHNbT3YdLfExc/u+Q57tR/EcLgovqICR4wU= dkliban@localhost.localdomain"
   tasks:
-    - name: Install RSSH
+    - name: "Install RSSH"
       package:
         name: rssh
         state: installed
-    - name: Add RSSH config for doc builder's user
-      lineinfile:
-        path: /etc/rssh.conf
-        state: present
-        regexp: '^user\s+=\s+{{ doc_user }}'
-        line: "user = {{ doc_user }}:011:10000:"
-    - name: Create doc builder's user
-      user:
-        name: "{{ doc_user }}"
-        shell: /usr/bin/rssh
-        groups:
-          - rsshusers
-        comment: "Doc Builder User"
+
     - name: "Create vhost for {{ doc_vhost }}"
       include_role:
         name: httpd
         tasks_from: vhost
       vars:
         website_domain: "{{ doc_vhost }}"
         document_root: "/var/www/{{ doc_vhost }}"
-        document_root_group: "{{ doc_user }}"
         use_tls: True
         use_letsencrypt: True
         force_tls: True
-    # rsync is already installed for the web builder sync
-    - name: Install doc builder's SSH keys
-      authorized_key:
-          key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDNfEAXdhT7mIf4d+XOWvE4hw5uduVyqc1N3ttqHV/5SM37XWHvln9D0IzMtDllXrmvlFuUA8sUCnc3zw3s+zIFh0ln+nhEQAo3StvchmlzmYJD1VyS1e8Pfd9xnD8DOBQNdy3aq4ZVrLtCAYndfX1Y5sfqlgN35OQ+7frjKaJXQWVHuLDGTxWR/y82EOT5JyXPyZd6B9VsQRPmB5oK1eSqMNQikuXJITDRxK6olfIdLzze85gV0aRJYo6GD5Aj68SciIsR95y//afwOQZr8m7OjygTsxLTlzsoRVR2N4uTyOe2i2szqy9kxUnvTiTVDULWicQZLBmv7S1pjaytKJnj pulp-doc-builder"
-          user: "{{ doc_user }}"
+        # TODO: enable when pulpcore is properly synced
+        #redirects:
+        #- src: '^/$'
+        #  target: '/pulpcore'
+        #  match: True
```

Comment:
Can we also add a redirect rule that redirects a requested URL that does not start with "/pulp_" or "/pulpcore" to "/pulpcore/{$url}" ?

Reply:
yes, will do that on Monday (it's late here ;-P)

Reply:
Thank you! Have a good weekend!

Reply:
@duck-rh How's this going?

Reply:
@dkliban I made the changed and it seems to work fine.